qa/L0_lifecycle/lifecycle_test.py

```diff
@@ -2916,6 +2917,189 @@ def test_concurrent_load_unload(self):
             for model_name in model_names:
                 self.assertEqual(is_load, triton_client.is_model_ready(model_name))
 
+    def test_concurrent_model_instance_load_speedup(self):
+        # Initialize client
+        try:
+            triton_client = httpclient.InferenceServerClient(
+                "localhost:8000", verbose=True
+            )
+        except Exception as ex:
+            self.assertTrue(False, "unexpected error {}".format(ex))
+        models = ["identity_fp32"]
```

Comment:
Maybe we can also add a Python model for testing?

Reply:
`identity_fp32` is a python model, `../python_models/identity_fp32`, I can rename it to be consistent with the other test.